pkg/odo/cli/version/version.go

```diff
@@ -132,6 +132,6 @@ func GetLatestReleaseInfo(info chan<- string) {
 			"to update manually, or visit https://github.com/openshift/odo/releases\n" +
 			"---\n" +
 			"If you wish to disable the update notifications, you can disable it by running\n" +
```

Comment:
Why not a simpler message like:
```
You can disable the update notifications by running 'odo preference set UpdateNotification false'
```
instead?

Reply:
I actually prefer the original message 😅